src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -194,6 +224,45 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetPureOrNotTracepoint(bool is_pure) {
+  ui_->tracepointCallstackCollectionCheckBox->setChecked(!is_pure);
+}
+
+bool CaptureOptionsDialog::GetPureOrNotTracepoint() const {
+  return !ui_->tracepointCallstackCollectionCheckBox->isChecked();
+}
+
+void CaptureOptionsDialog::SetTracepointCallstackMethod(
+    CaptureOptions::TracepointCallstackMethod tracepoint_callstack_method) {
+  switch (tracepoint_callstack_method) {
+    case CaptureOptions::kFramePointersTracepoint:
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(false);
+      break;
+    case CaptureOptions::kDWARFTracepoint:
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(false);
+      break;
+    default:
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(false);
+      break;
+  }
+}
+
+orbit_grpc_protos::CaptureOptions::TracepointCallstackMethod
+CaptureOptionsDialog::GetTracepointCallstackMethod() const {
+  if (ui_->tracepointCallstackFramepointersMethodRadioButton->isChecked()) {
+    ORBIT_CHECK(!ui_->tracepointCallstackDWARFMethodRadioButton->isChecked());
```

Comment:
I don't think these CHECKs should be here. Nothing bad is gonna happen if they fail.

Reply:
Done.